apps/website/src/examples/Breadcrumbs.customOverflowBackButton.tsx

```diff
@@ -0,0 +1,37 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+import * as React from 'react';
+import { Breadcrumbs, Button, Tooltip, IconButton } from '@itwin/itwinui-react';
+import { SvgMore } from '@itwin/itwinui-icons-react';
+
+export default () => {
+  const items = Array(10)
+    .fill(null)
+    .map((_, index) => <Button key={index}>Item {index}</Button>);
+
+  return (
+    <div style={{ maxWidth: '50%', border: '1px solid lightpink', padding: 8 }}>
```

Comment:
a bigger maxWidth value will work better on smaller screens

Reply:
Updated to match the overflow example already there

Reply:
it's better but still doesn't look great on small screen widths
![image](https://user-images.githubusercontent.com/9084735/227009855-cb38b1f4-cb56-4928-91d9-9824aac9150f.png)

the same problem is also present in the original example
![image](https://user-images.githubusercontent.com/9084735/227009756-09635a19-2c59-4a28-976a-f8b7a54088a3.png)

you can guard against this using something like `maxWidth: 'min(90%, 425px)'`, or just a larger percentage in this case (`maxWidth: 70%`)

Reply:
Updated the truncation examples: 
![image](https://user-images.githubusercontent.com/37936169/227625763-95b2c3aa-eb06-4145-910f-7a0751a61fe8.png)